contracts/BimaFaucet.sol

```diff
@@ -1,24 +1,35 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.19;
+pragma solidity 0.8.20;
 
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
 contract BimaFaucet is Ownable {
-  uint256 public constant TOKEN_AMOUNT = 2e18;
+  uint256 public tokenAmount = 2e18;
   uint256 public constant INTERVAL = 1 days;
 
   mapping(address => mapping(address => uint256)) public tokensRecievedAt; // account => token => timestamp
 
+  constructor() Ownable(msg.sender) {}
```

Comment:
I don't think it's necessary?

I wrote tests for it in `Faucet.test.ts` and everything works normally without this.